manual/cops_lint.md

```diff
@@ -2331,7 +2331,7 @@ require 'unloaded_feature'
 
 Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
 --- | --- | --- | --- | ---
-Enabled | Yes | Yes  | 0.43 | -
+Enabled | Yes | Yes  | 0.43 | 0.74.0
```

Comment:
CI has failed. Can you run `rake generate_cops_documentation` and apply it?
https://circleci.com/gh/rubocop-hq/rubocop/62613